fbw-common/src/wasm/systems/systems/src/shared/mod.rs

```diff
@@ -815,6 +817,71 @@ impl From<MachNumber> for f64 {
     }
 }
 
+impl MaxDifference for MachNumber {
+    fn max_diff(self, other: Self) -> f64 {
+        (f64::from(self) - f64::from(other)).abs()
+    }
+}
+
+impl MachNumber {
+    // All formulas from Jet Transport Performance Methods by Boeing (March 2009 revision)
+
+    /// Get the pressure ratio for a given pressure
+    fn delta(air_pressure: Pressure) -> f64 {
+        air_pressure.get::<hectopascal>() / 1013.25
+    }
+
+    /// Get the temperature ratio for a given temperature
+    fn theta(temperature: ThermodynamicTemperature) -> f64 {
+        temperature.get::<kelvin>() / 288.15
+    }
+
+    /// Convert the mach number to a calibrated airspeed for a given atmospheric pressure.
+    pub fn to_cas(self, air_pressure: Pressure) -> Velocity {
+        Velocity::new::<knot>(
+            1479.1
```

Comment:
Is it possible to extract this value to a constant to have a more descriptive name for it?

Reply:
The constant is a combination of some constant terms that doesn't really have a handy name. I don't think anyone should ever alter these equations so I don't expect it to be a problem. Instead they can refer to the source material noted in the comment above (Jet Transport Performance Methods) which provides a full derivation (many pages).